CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # OHHTTPStubs — CHANGELOG
 
-## [8.0.0](https://github.com/AliSoftware/OHHTTPStubs/releases/tag/7.0.0)
+
+## [8.1.0](https://github.com/AliSoftware/OHHTTPStubs/releases/tag/8.1.0)
+
+* Added support for Swift Package Manager and an example project.    
```

Comment:
Since this will be a breaking change with the class rename, it will have to be a 9.0.0 rather than a 8.1.0

Reply:
I originally thought the class rename was good thing, but after creating it I'm not so sure anymore.  I created that as a second PR, because it's a **HUGE** change.  It also feels a bit weird to `import OHTTPSStubs` everywhere but then use `HTTPStubs` everywhere.  

I'm honestly not sure about the right path forward, but leaning towards this PR without the renaming.